draft-ietf-quic-transport.md

```diff
@@ -1684,7 +1686,9 @@ retry_source_connection_id transport parameter.
 
 The values provided by a peer for these transport parameters MUST match the
 values that an endpoint used in the Destination and Source Connection ID fields
-of Initial packets that it sent. Including connection ID values in transport
+of Initial packets that it sent (and received, for servers). Endpoints MUST
+validate that received transport parameters match received Connection ID values.
+Including connection ID values in transport
```

Comment:
I'm calling this "editorial" even though it adds a new MUST, because I believe the intent was already that you do this, and so the text is just reiterating an existing expectation.